frontend/src/concepts/pipelines/content/tables/pipelineRunJob/PipelineRunJobTableRow.tsx

```diff
@@ -32,7 +32,9 @@ const PipelineRunJobTableRow: React.FC<PipelineRunJobTableRowProps> = ({
     <Tr>
       <CheckboxTd id={job.id} isChecked={isChecked} onToggle={onToggleCheck} />
       <Td>
-        <TableRowTitleDescription title={job.name} description={job.description} />
+        <TableText wrapModifier="truncate">
+          <TableRowTitleDescription title={job.name} description={job.description} />
+        </TableText>
```

Comment:
Does `<Td modifier="truncate">` work here?

Reply:
I don't think so. Because the truncated part will be hidden like this. 

![Screenshot from 2023-12-01 13-25-27](https://github.com/opendatahub-io/odh-dashboard/assets/99261071/597aff35-7ede-4756-a1a0-e1c380d53a1c)